src/components/OutputForm.tsx

```diff
@@ -144,5 +145,12 @@ export const OutputTime = ({ label, time, relative }: { label: string; time?: nu
     };
   });
 
-  return <OutputField label={label}>{time && <RelativeTimeText time={time} baseTime={nowTime} defaultToTime={!relative} />}</OutputField>;
+  let useRelative = relative ?? false;
```

Comment:
This seems unnecessary, `undefined` is already `falsey`

Reply:
Switched to an enum